src/poliastro/constants.py

```diff
@@ -0,0 +1,111 @@
+"""Astronomical and physics constants in SI units.
+
+This module complements constants defined in `astropy.constants`
+
+Unless otherwise specified, gravitational and mass parameters were obtained from:
+
+* Archinal, B. A. et al. “Report of the IAU Working Group on Cartographic Coordinates and Rotational Elements: 2009.”
+  Celestial Mechanics and Dynamical Astronomy 109.2 (2010): 101–135. Crossref. Web. `DOI: 10.1007/s10569-010-9320-4`_
+
+and radii were obtained from:
+
+* Luzum, Brian et al. “The IAU 2009 System of Astronomical Constants: The Report of the IAU Working Group on Numerical
+  Standards for Fundamental Astronomy.” Celestial Mechanics and Dynamical Astronomy 110.4 (2011): 293–304.
+  Crossref. Web. `DOI: 10.1007/s10569-011-9352-4`_
+
+
+.. _`DOI: 10.1007/s10569-010-9320-4`: http://dx.doi.org/10.1007/s10569-010-9320-4
+.. _`DOI: 10.1007/s10569-011-9352-4`: http://dx.doi.org/10.1007/s10569-011-9352-4
+
+"""
+
+from astropy.constants import Constant
+
+
+G = Constant('G', 'Constant of gravitation', 6.67428e-11, 'm3 / (kg * s2)', 6.7e-15,
+             'IAU 2009 system of astronomical constants', system='si')
+
+GM_sun = Constant('GM_sun', 'Heliocentric gravitational constant', 1.32712442099e20, 'm3 / (s2)', 1.0e10,
```

Comment:
In Astropy they use the same exponent for the uncertainty. I think it's a good practice that we can keep.